scripts/reindex.py

```diff
@@ -116,6 +122,21 @@ def main(argv: List[str]):
             azul_client.remote_reindex(args.partition_prefix_length)
         else:
             azul_client.reindex()
+        if not args.nowait:
+            logger.info('Waiting for queues to empty ... ')
```

Comment:
This was copied from the IT. I'll give you one guess as to what I might object to here.

Reply:
Refactored duplicated code into a static method in Queues class.